guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -42,12 +42,31 @@ endif::[]
 
 In {Project} provisioning, the PXE loader option defines the DHCP `filename` option to use during provisioning.
 For BIOS systems, use the *PXELinux BIOS* option to enable a provisioned node to download the `pxelinux.0` file over TFTP.
-For UEFI systems, use the *PXEGrub2 UEFI* option to enable a TFTP client to download `grub2/grubx64.efi` file, or use the *PXEGrub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` from {SmartProxy} with the HTTP Boot feature.
+For UEFI systems, use the *Grub2 UEFI* option to enable a TFTP client to download `grub2/grubx64.efi` file, or use the *Grub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` from {SmartProxy} with the HTTP Boot feature.
```

Comment:
I am not sure what "use the <option>" means. Is the user responding to CLI prompts? Or is this a UI? 

It might be better to use "Select", depending on the context.

Reply:
`PXEGrub2` is the internal identifier of the PXE laoder. But if you have a look at the Web UI e.g. in `Hosts` > `Create Hosts` > `Operating System` > `PXE loader` in this list a "display name" is used. And that is `Grub2 UEFI` resp. `Grub2 UEFI HTTP`. In the documentation we should use the names visible to users in the Web UI.

Reply:
Regarding "use the" versus "select": I would prefer "select", as it's in the Web UI and you select the PXE loader from a dropdown selection.

Reply:
"use the" => "select the" :heavy_check_mark: